about.html

```diff
@@ -273,7 +273,7 @@ <h5><b>Digital Inclusion Global Challenge</b></h5>
               <div class="col-md-3 col-sm-6 col-xs-12">
                   <div class="team-member">
                       <img class="img-circle our-awards" src="images/awards/uok.png" alt="biz minds award image">
```

Comment:
Provide the `a href` tag to the images too. So the images must be clickable and it needs to redirect to the links.